python/src/opendp/mod.py

```diff
@@ -258,6 +260,126 @@ def __iter__(self):
         # this overrides the implementation of __iter__ on POINTER, 
         # which yields infinitely on zero-sized types
         raise ValueError("Measurement does not support iteration")  # pragma: no cover
+    
+
+class Odometer(ctypes.POINTER(AnyOdometer)): # type: ignore[misc]
+    """A differentially private unit of computation with no privacy limit.
+    An odometer contains a function that returns a queryable with a function and a relation.
+    Differentially private queries may be passed to the queryable,
+    as well as queries to check the current privacy usage.
+
+    :example:
+
+    >>> import opendp.prelude as dp
+    >>> dp.enable_features("contrib")
+    ...
+    >>> base_rr = dp.m.make_randomized_response_bool(prob=0.6)
+    ...
+    >>> # create an Odometer representing a fully adaptive compositor
+    >>> o_comp = dp.c.make_fully_adaptive_composition(
+    ...     input_domain=base_rr.input_domain,
+    ...     input_metric=base_rr.input_metric,
+    ...     output_measure=base_rr.output_measure,
+    ...     d_in=1,
+    ... )
+    ...
+    >>> # invoke the odometer to get a queryable
+    >>> qbl_comp = o_comp(True)
```

Comment:
This corresponds to `__call__` below? I'm not sure what the `True` does, or if we'd every use anything else.

Reply:
In this case, the `True` is the entire dataset.